source/ranges.tex

```diff
@@ -302,170 +325,191 @@
   }
 
   // \ref{range.adjacent.transform}, adjacent transform view
   template<@\libconcept{forward_range}@ V, @\libconcept{copy_constructible}@ F, size_t N>
     requires @\seebelow@
-  class adjacent_transform_view;
+  class adjacent_transform_view;                                                    // freestanding
 
   namespace views {
     template<size_t N>
-      inline constexpr @\unspec@ adjacent_transform = @\unspec@;
-    inline constexpr auto @\libmember{pairwise_transform}{views}@ = adjacent_transform<2>;
+      inline constexpr @\unspec@ adjacent_transform = @\unspec@;                // freestanding
+    inline constexpr auto @\libmember{pairwise_transform}{views}@ = adjacent_transform<2>;               // freestanding
   }
 
   // \ref{range.chunk}, chunk view
   template<@\libconcept{view}@ V>
     requires @\libconcept{input_range}@<V>
-  class chunk_view;
+  class chunk_view;                                                                 // freestanding
 
   template<@\libconcept{view}@ V>
     requires @\libconcept{forward_range}@<V>
-  class chunk_view<V>;
+  class chunk_view<V>;                                                              // freestanding
 
   template<class V>
-    inline constexpr bool enable_borrowed_range<chunk_view<V>> =
+    inline constexpr bool enable_borrowed_range<chunk_view<V>> =                    // freestanding
       @\libconcept{forward_range}@<V> && enable_borrowed_range<V>;
 
-  namespace views { inline constexpr @\unspec@ chunk = @\unspec@; }
+  namespace views {
+    inline constexpr @\unspec@ chunk = @\unspec@;                               // freestanding
+  }
 
   // \ref{range.slide}, slide view
   template<@\libconcept{forward_range}@ V>
     requires @\libconcept{view}@<V>
-  class slide_view;
+  class slide_view;                                                                 // freestanding
 
   template<class V>
     inline constexpr bool enable_borrowed_range<slide_view<V>> =
-      enable_borrowed_range<V>;
+      enable_borrowed_range<V>;                                                     // freestanding
 
-  namespace views { inline constexpr @\unspec@ slide = @\unspec@; }
+  namespace views {
+    inline constexpr @\unspec@ slide = @\unspec@;                               // freestanding
+  }
 
   // \ref{range.chunk.by}, chunk by view
   template<@\libconcept{forward_range}@ V, @\libconcept{indirect_binary_predicate}@<iterator_t<V>, iterator_t<V>> Pred>
     requires @\libconcept{view}@<V> && is_object_v<Pred>
-  class chunk_by_view;
+  class chunk_by_view;                                                              // freestanding
 
-  namespace views { inline constexpr @\unspec@ chunk_by = @\unspec@; }
+  namespace views {
+    inline constexpr @\unspec@ chunk_by = @\unspec@;                            // freestanding
+  }
 }
 
 namespace std {
   namespace views = ranges::views;
```

Comment:
Ignore this. A namespace alias is not an entity.

Reply:
fixed

Reply:
"Fixed" is ambiguous. I thought you did apply the suggestion. Thankfully, I was wrong.

Reply:
fixed again :)

Reply:
Actually, the instructions for `<ranges>` reads
>  Please append a `// freestanding` comment to every entity in `<ranges>` except for the following entities:

And `views` is not an entity, so it needs to be reverted.
I suppose this begs the question, do freestanding implementations have to use `std::ranges::views` rather than `std::views`?

Reply:
Opened https://github.com/cplusplus/draft/issues/5724 for this.